addons/web_editor/static/src/js/editor/transcoder.js

```diff
@@ -105,25 +105,42 @@ function getMatchedCSSRules(a) {
         delete style.display;
     }
 
-    _.each(['margin', 'padding'], function (p) {
-        if (style[p+'-top'] || style[p+'-right'] || style[p+'-bottom'] || style[p+'-left']) {
-            if (style[p+'-top'] === style[p+'-right'] && style[p+'-top'] === style[p+'-bottom'] && style[p+'-top'] === style[p+'-left']) {
+    _.each([['margin'], ['padding'], ['border', 'style']], function (attr) {
```

Comment:
Commit message should explain this and inline comments should explain this new transcoding behavior